src/containerapp/azext_containerapp/_help.py

```diff
@@ -1271,3 +1271,42 @@
               --environment MyContainerappEnv \\
               --compose-file-path "path/to/docker-compose.yml"
 """
+
+# Patch commands
+helps['containerapp patch'] = """
+    type: group
+    short-summary: Patch Azure Container Apps. Patching is only available for the apps built using the source to cloud feature. See https://aka.ms/aca-local-source-to-cloud
+"""
+
+helps['containerapp patch list'] = """
+    type: command
+    short-summary: List container apps that can be patched. Patching is only available for the apps built using the source to cloud feature. See https://aka.ms/aca-local-source-to-cloud
+    examples:
+    - name: List container apps that can be patched.
+      text: |
+          az containerapp patch list -g MyResourceGroup --environment MyContainerAppEnv
+    - name: List patchable and unpatchable container apps.
+      text: |
+          az containerapp patch list -g MyResourceGroup --environment MyContainerAppEnv --show-all
+"""
+
+helps['containerapp patch apply'] = """
+    type: command
+    short-summary: List and apply container apps to be patched. Patching is only available for the apps built using the source to cloud feature. See https://aka.ms/aca-local-source-to-cloud
+    examples:
+    - name: List container apps that can be patched and apply patch.
+      text: |
+          az containerapp patch apply -g MyResourceGroup --environment MyContainerAppEnv
+"""
+
+helps['containerapp patch interactive'] = """
```

Comment:
We sort of expect some end users to use the interactive CLI command. By referring them to a separative command, there's less confusion for them to run into situation where they accidentally use `patch apply` command without `--interactive` and automatically patch all the images they have. 

p.s.: the `patch apply` command is for a more system-friendly way to call the patcher and patch all the containers that it deems patchable.

Reply:
We should double-check with the Azure CLI folks about the flow users expect here, but from my experience working on Azure PowerShell, if there is an "interactive" (_aka_, prompted) flow for the user, it will be the default behavior of a command, and any need by the user to skip this prompt would be done by a `--force`/`-f` flag.

In my opinion, it's an antipattern to have two commands that do the exact same thing, but the only difference between the two is whether or not you're prompted to begin the patching process -- it makes the most sense if these were bundled together and distinguished by a `--force`/`-f` flag.

Reply:
@harryli0108 @cormacpayne Let's sync about this tomorrow, but I wouldn't treat this as a release blocker. Resolving the comment for now.